demos/kfp/docling/asr-conversion/conversion-on-gpu/docling_asr_convert_pipeline.py

```
@@ -0,0 +1,550 @@
+        base_url=base_url,
+        audio_filenames=audio_filenames,
+    )
+    import_task.set_caching_options(True)
+
+    audio_splits = create_audio_splits(
+        input_path=import_task.output,
+        num_splits=num_workers,
+    ).set_caching_options(True)
+
+    with dsl.ParallelFor(audio_splits.output) as audio_split:
+        with dsl.If(use_gpu == True):
```

Flake8 (7.2.0)

[error] comparison to True should be 'if cond is True:' or 'if cond:'

(E712)